wp-tevko-responsive-images.php

```diff
@@ -294,6 +294,38 @@ function tevkori_filter_content_images( $content ) {
 	$uploads_dir = wp_upload_dir();
 	$path_to_upload_dir = $uploads_dir['baseurl'];
 
+	preg_match_all( '|<img ([^>]+' . $path_to_upload_dir . '[^>]+)[\s?][\/?]>|i', $content, $matches );
+
+	$images = $matches[0];
+	$ids = array();
+
+	foreach( $images as $image ) {
+		if ( preg_match( '/wp-image-([0-9]+)/i', $image, $class_id ) ) {
+			(int) $id = $class_id[1];
+			if( $id ) {
+				$ids[] = $id;
+			}
+		}
+	}
+
+	if ( 0 < count( $ids ) ) {
```

Comment:
`WP_Query` makes more than one call to the DB, increasing the minimum number if images required before warming the cache may result in fewer DB calls.